app/views/layouts/main.hbs

```diff
@@ -65,17 +70,5 @@
   </body>
 
   <footer>
-    <script src="/js/vendor/handlebars.runtime.min.js"></script>
```

Comment:
That said, I'd also love to drop handlebars. I don't think I totally understand how using handlebars makes it difficult to use some of angular's more interesting functionality, but as I mentioned before, I don't need to understand. We can chat more about it in person.